GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -164,11 +164,29 @@ private void AddExcludePath_Click(object sender, EventArgs e)
 
             if (path is not null)
             {
-                path = path.Replace(Module.WorkingDir, "");
+                path = MakePathRelativeToModule(path);
                 textBoxExcludePaths.Text += path;
             }
         }
 
+        private string MakePathRelativeToModule(string targetPath)
+        {
+            targetPath = targetPath.Replace(Module.WorkingDir, "").ToPosixPath();
```

Comment:
❔ 
```suggestion
            targetPath = targetPath.ToPosixPath().Replace(Module.WorkingDir, "");
```
This should be tested, too.

Reply:
Tested, this doesn't work since WorkingDir is not stored as Posix path

Reply:
Does the user need to input paths in the native form in order to match WorkingDir?
I would not make assumptions on the type of path separators in WorkingDir either.
I.e. convert both?

Reply:
```suggestion
            targetPath = targetPath.ToPosixPath().Replace(Module.WorkingDir.ToPosixPath(), "");
```